modules/internal/ChapelBase.chpl

```diff
@@ -1264,17 +1264,18 @@ module ChapelBase {
     }
   }
 
-  pragma "task complete impl fn"
-  extern proc chpl_comm_task_end(): void;
+  extern proc chpl_comm_unordered_consist_fence(): void;
```

Comment:
It might need `  pragma "task complete impl fn"` for the forall unordered opt to work right. We can add another flag if that turns out to be the case.

Reply:
I looked at how the compiler used this and the best I could tell, the `pragma "task complete impl fn"` actually did seem to have to do with task termination rather than unordered operations.  Testing seems to indicate (so far) that unordered ops are getting completed and made consistent properly.